test/functional/apps/dashboard/embeddable_library.ts

```diff
@@ -68,44 +68,5 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       );
       expect(libraryActionExists).to.be(true);
     });
-
-    it('unlink map panel from embeddable library', async () => {
```

Comment:
Not necessarily related to this PR - because these tests are associated with the legacy maps, but it seems like there isn't any functional test coverage for linking / unlinking maps panels from the embeddable. Perhaps tests similar to these should be added to `x-pack/test/functional/apps/dashboard/dashboard_maps_by_value.ts`